Documentation/design/resource_dep.dot

```diff
@@ -0,0 +1,121 @@
+digraph resource {
+  size="8,6"; ratio=fill; node[fontsize=20];
+  compound=true;
+  node [shape=rectangle];
+
+  {
+    node [shape=plaintext];
+    init -> render -> prepare -> launch;
+  }
+
+  // User inputs.
+  subgraph cluster_input {
+    node [shape=rectangle,style=filled,color=gainsboro];
+    base_domain [label="base domain"];
+    cluster_name [label="cluster name"];
+    license [label="license"];
+    pull_secret [label="pull secret"];
+    platform [label="platform"];
+    email_address [label="email address"];
+    password [label="password"];
+  }
+
+  subgraph cluster_hint {
+    style=invis;
+    node [shape=rectangle,style=filled,color=lightblue];
+    output [label="output"];
+    node [shape=rectangle,style=filled,color=gainsboro];
+    input [label="input "];
+  }
+
+  // Install config.
+  {
+    node [style=filled,color=sandybrown];
+    install_config [label="install-config.yaml"];
+  }
+
+  // TLS assets
+  {
+    node [style=filled,color=violet];
+    tls_assets [label="tls assets"];
+  }
+
+  // Operator assets
+  subgraph cluster_operator {
+    style=invis;
+    node [style=filled,color=lightgreen];
+    operator_configs [label="operator configs"];
+    operator_manifests [label="operator manifests"];
+  }
+
+  // Cluster assets
+  subgraph cluster_cluster_assets {
+    style=invis;
+    node [style=filled, color=lightpink];
+    bootstrap_control_plane [label="bootstrap control plane manifests"];
+    kubernetes_control_plane [label="kubernetes control plane manifests"];
+  }
+
+  // Ignition files
+  subgraph cluster_ignition {
+    style=invis;
+    node [style=filled, color=gold];
+    bootstrap_ignition [label="bootstrap.ign"];
+    master_ignition [label="master.ign"];
+    worker_ignition [label="worker.ign"];
+  }
+
+  // Output after cluster launch
+  {
+
+    node [style=filled,color=lightblue];
+    terraform_state [label="*.tfstate"];
+
+    subgraph cluster_kubeconfig {
+      style=invis;
+      kubelet_kubeconfig [label="kubeconfig-kubelet"];
+      kubeconfig [label="kubeconfig"];
+    }
+  }
+
+  // Dependencies
+  pull_secret -> install_config [ltail=cluster_input];
+
+  install_config -> tls_assets;
+  install_config -> operator_manifests;
+  install_config -> operator_configs;
+
+  tls_assets -> kubeconfig;
+  tls_assets -> kubelet_kubeconfig;
+  tls_assets -> kubernetes_control_plane;
+  tls_assets -> bootstrap_control_plane;
+
+  tls_assets -> bootstrap_ignition;
+  tls_assets -> master_ignition;
+  tls_assets -> worker_ignition;
+
+  operator_configs -> bootstrap_control_plane;
```

Comment:
Why is this a dependency?

Reply:
we will launch the operators on the bootstrap node, so we need their configs too.